src/App/Router.php

```diff
@@ -111,6 +111,7 @@ public function collectRoutes()
 		});
 		$this->routeCollector->addRoute(['GET'],         '/filer[/{id:\d+}]',    Module\Filer\SaveTag::class);
 		$this->routeCollector->addRoute(['GET'],         '/filerm/{id:\d+}',     Module\Filer\RemoveTag::class);
+		$this->routeCollector->addRoute(['GET', 'POST'], '/follow_confirm',      Module\FollowConfirm::class);
```

Comment:
I don't see either a `content` nor a `rawContent`, so this would be:
```suggestion
		$this->routeCollector->addRoute(['GET']        , '/follow_confirm',      Module\FollowConfirm::class);
```

Reply:
With only "post" it hadn't worked. Possibly this is a bug in the router.

Reply:
@MrPetovan Ping.. do you know this problem?

Reply:
Yes.

https://github.com/friendica/friendica/blob/097ef9b326f74779b4f14774109f1074ddc64891/src/App/Router.php#L33-L34

Reply:
Arg.. ^^ .. good 2 know